tests/test_issues/test_issue_571_sql_java.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from linkml_runtime.linkml_model import SchemaDefinition, SlotDefinition, ClassDefinition
+from linkml_runtime.dumpers import yaml_dumper
+from linkml.generators.yamlgen import YAMLGenerator
+from linkml.generators.sqlddlgen import SQLDDLGenerator
+from linkml.generators.javagen import JavaGenerator
+from linkml.generators.projectgen import ProjectGenerator
+
+# import os
+# from contextlib import redirect_stdout
+# from tests.test_generators.environment import env
+
+# SCHEMA = env.input_path('kitchen_sink.yaml')
+# JAVA_DIR = env.expected_path('kitchen_sink_java')
+PACKAGE = 'org.sink.kitchen'
+
+
+class BasicExample(unittest.TestCase):
+    """An illustration of failed java generation will go here.
+    Right now I'm just getting used to unittest"""
+
+    # def test_valid_math(self):
+    #     self.assertEqual(1, 2 / 2)
+
+    # def test_bogus_math(self):
+    #     self.assertEqual(1, 0.5 + 0.501)
+
+    def test_sql_ddl_gen(self):
+        # todo make the schema creation a fixture and split out the different generations
+        schema_id = 'http://example.com/issue_571_schema'
+        issue_571_schema = SchemaDefinition(name="issue_571_schema", id=schema_id)
```

Comment:
a trifle wordy but YMMV!

Reply:
> a trifle wordy but YMMV!

I'm trying to figure how to use the kitchen sink schema as input